scripts/createDB.sql

```diff
@@ -25,6 +25,16 @@ CREATE TABLE `soen_390_db`.`user` (
 
   collate = utf8mb4_unicode_ci;
 
+-- Table to store the customers.
+CREATE TABLE `soen_390_db`.`customer` (
+  `customerID` INT NOT NULL AUTO_INCREMENT,
```

Comment:
Watch out for the way you case ids. In ``customerID``, both letters are uppercased, while on ``goodId``, you used camelcase. It'd be good to settle on one casing.